mockrxandroidble/src/main/java/com/polidea/rxandroidble/mockrxandroidble/RxBleConnectionMock.java

```diff
@@ -468,4 +473,9 @@ public Boolean call(byte[] ignored) {
         }
 
     }
+
+    @Override
+    public <T> Observable<T> queue(Func3<Scheduler, BluetoothGatt, RxBleGattCallback, RxBleRadioOperation<T>> builder) {
+        return Observable.never();
```

Comment:
Mocking both `BluetoothGatt` and `RxBleGattCallback` should be totally doable. But adding proper support for it would take some time - I am not even sure how many users the mock library has to judge if it is worth the effort or not.
`UnsupportedOperationException` should be fine for now.